perllib/FixMyStreet/Map/OSM.pm

```diff
@@ -57,6 +57,7 @@ sub display_map {
         if defined $c->get_param('lat');
     $params{longitude} = Utils::truncate_coordinate($c->get_param('lon') + 0)
         if defined $c->get_param('lon');
+    $params{zoomToBounds} = !defined $c->get_param('zoom');
```

Comment:
This can be a bit odd on an around page - whilst if `js=1` the pins are presumably within the viewport and so zooming to their extent doesn't do anything, it's not needed, and if `js=0` pins might be out of the viewport so your map ends up centred somewhere else (otoh, better for the pins loaded, but bit odd if you're expecting a particular centre). Perhaps this should be if `any_zoom` *and* not defined zoom parameter? Or was there a reason you removed `any_zoom` here.

(Aside: I think there's some long-standing interaction with OL's ArgParser that is looking for lat/lon in the URL and updating map centre if it sees them (defaulting to zoom 0), so e.g. `/around?pc=foo` will use default calculated zoom level, but `/around?lat=same&lon=location-as-foo` will go to zoom 0. Hmm.)